libs/Daux.php

```diff
@@ -90,6 +93,18 @@ public function initializeConfiguration($override_file = 'config.json')
         }
     }
 
+    public function setThemesPath($path)
+    {
+        $this->themes_path = $path;
+        if (!is_dir($this->themes_path) &&
+            !is_dir($this->themes_path = $this->local_base . DIRECTORY_SEPARATOR . $this->themes_path)
+        ) {
+            throw new Exception('The Themes directory does not exist. Check the path again : ' . $this->themes_path);
+        }
+        $this->options['themes_path'] = $this->themes_path;
+        $this->options['templates'] = $this->themes_path . DIRECTORY_SEPARATOR . 'core' . DIRECTORY_SEPARATOR . 'templates';
```

Comment:
No it's not the same purpose, one is given aside from the `.phar` archive, and the other is automatically updated.

When a user starts to roll out his own theme, he will disconnect himself from the current layout and when we release an update he will not update his own branch.

For the theme, I would even call it `daux`, so it will be able to contain the variants and that would reduce the default themes to only one folder without duplicate files (fonts, common css ...)